app/models/miq_widget/rss_content.rb

```diff
@@ -24,7 +24,7 @@ def generate(user_or_group)
 
   def internal_feed(user_or_group)
     resource.options[:limit_to_count] = widget_options[:row_count] || 5
-    SimpleRSS.parse(resource.generate(nil, nil, nil, user_or_group))
+    SimpleRSS.parse({:text => resource.generate(nil, nil, user_or_group), :content_type => Mime[:rss]})
```

Comment:
I understand where this change comes from, but would expect `SimpleRSS.parse` to accept a string.

Reply:
https://github.com/cardmagic/simple-rss/blob/master/lib/simple-rss.rb#L71

https://github.com/cardmagic/simple-rss/blob/master/lib/simple-rss.rb#L40

Reply:
Correct. Fixed.